geometry/Overview.bs

```diff
@@ -890,13 +890,13 @@ means to run the following steps. It will either return a <a>4x4 abstract matrix
     for the CSS 'transform' property. The result will be a <<transform-list>>, the keyword
     ''transform/none'', or failure. If <var>parsedValue</var> is failure, or any <<transform-function>>
     has <<length>> values without <a spec=css-values>absolute length</a> units, or any keyword other
-    than ''transform/none'' is used, then return failure. [[!CSS3-SYNTAX]] [[!CSS3-TRANSFORMS]]
+    than ''transform/none'' is used, then return failure. [[!CSS3-SYNTAX]] [[!CSS3-TRANSFORMS]] [[!CSS3-TRANSFORMS-2]]
 3. If <var>parsedValue</var> is ''transform/none'', set <var>parsedValue</var> to a
     <<transform-list>> containing a single identity matrix.
-4. Let <var>2dTransform</var> track the 2D/3D dimension status of <var>parsedValue</var>.
+4. Let <var>2dTransform</var> track whether <var>parsedValue</var> is a 2D or 3D transform.
```

Comment:
This one unrelated change: the phrase "the 2D/3D dimension status" didn't make much sense to me, so I suggested changing it to a simpler phrase. Is this phrase necessary (i.e. used in other places)? If so, I can undo this specific change and save it for a separate change that makes the wording simpler in all locations.